umm-spec-lib/test/cmr/umm_spec/test/migration/version/core.clj

```diff
@@ -1,17 +1,17 @@
-(ns cmr.umm-spec.test.migration.version.variable
+(ns cmr.umm-spec.test.migration.version.core
   (:require
-   [clojure.test :refer :all]
+   [clojure.test :refer [deftest is]]
    [cmr.umm-spec.migration.version.core :as vm]
    [cmr.umm-spec.versioning :as v]))
 
 
 (deftest test-version-steps
   (with-bindings {#'v/versions {:collection ["1.14" "1.15" "1.15.1" "1.15.2"
-                                             "1.15.3" "1.15.4" "1.15.5" "1.16"]}}
-    (is (= [] (#'vm/version-steps :variable "1.14" "1.14")))
```

Comment:
why this change?

Reply:
Originally this namespace had a bad namespace name - so this test never ran. 
When I ran it, it didn't work.  The with-bindings changes the vector of the collection from its original definition.  Since the definition is for collections the :variable var didn't return anything, so I had to change this test to get it to run and then pass.